po/zh_CN.po

```diff
@@ -3070,48 +3109,48 @@ msgstr ""
 #: gsumm.c:66
 #, c-format
 msgid "o has length %d but sum(l)=%d"
-msgstr "o �ĳ���Ϊ %d���� sum(l) = %d"
+msgstr "o 的长度为 %d，但 sum(l) = %d"
 
 #: gsumm.c:69
 msgid "Internal error: o's maxgrpn attribute mismatches recalculated maxgrpn"
-msgstr "�ڲ�����o �� maxgrpn ���������¼���� maxgrpn ��ƥ��"
+msgstr "内部错误：o 的 maxgrpn 属性与重新计算的 maxgrpn 不匹配"
 
 #: gsumm.c:89
 #, c-format
 msgid ""
 "Internal error: nrow=%d  ngrp=%d  nbit=%d  shift=%d  highSize=%d  nBatch=%d  "
 "batchSize=%d  lastBatchSize=%d\n"
-msgstr "�ڲ�����nrow=%d ngrp=%d  nbit=%d  shift=%d  highSize=%d  nBatch=%d  "
+msgstr "内部错误：nrow=%d ngrp=%d  nbit=%d  shift=%d  highSize=%d  nBatch=%d  "
 "batchSize=%d  lastBatchSize=%d\n"
 
 #: gsumm.c:98
 #, c-format
 msgid "gforce initial population of grp took %.3f\n"
-msgstr "grp �� gforce ��ʼȺ����ռ�� %.3f\n "
+msgstr "grp 的 gforce 初始群体数占了 %.3f\n "
```

Comment:
grp 的 gforce 初始群体占了 %.3f\n

Reply:
Line 311 and below are outside of my assignment. Did not know why there are diffs here because I did not touch them at all. Will these comments be responded by other  collaborators and I can safely ignore review comments below?

Reply:
Maybe the difference of encoding or different CR/LF between windows and linux. Please ignore them. Though we do need some better approach to avoid this kind of situation, otherwise if you merge directly that will change their parts.
@MichaelChirico This may need your attention.

Reply:
Yes this is from Chun-Hui's section. There must be some encoding issue... hard to tell from the web GUI.

I don't really know a good solution. Will keep an eye on what happens upon merging this...